src/jaxoplanet/experimental/starry/light_curves.py

```diff
@@ -76,7 +90,11 @@ def map_light_curve(map, r: float, xo: float, yo: float, zo: float, theta: float
     Returns:
         ArrayLike: light curve
     """
-    U = jnp.array([1, *map.u])
+    if map is None:
+        return jnp.zeros_like(theta)
+
+    # TODO(lgrcia): Is this the right behavior when map.u is None?
+    U = jnp.array([1, *(() if map.u is None else map.u)])
```

Comment:
@lgrcia Can you check this behavior for `map` is `None` and `map.u` is `None`?

Reply:
I think the behavior is ok for these cases. I played with

```python
import jax
import numpy as np
from jaxoplanet.experimental.starry import light_curves, Map

y = np.linspace(-2, 2, 100)

map = Map(y=None, period=1.2, u=None)

flux = jax.vmap(light_curves.map_light_curve, in_axes=(None, None, None, 0, None, None))(
    map, 0.1, 0.0, y, 2.0, 0.0
)
```

Let me know if you think about different tests.